ru/mail/chrome/messenger/accountCreation.dtd

```diff
@@ -1,15 +1,15 @@
-<!ENTITY autoconfigWizard.title          "Настройка учётной записи почты">
+﻿<!ENTITY autoconfigWizard.title          "Настройка учетной записи почты">
 <!ENTITY name.accesskey                  "n">
-<!ENTITY name.placeholder                  "First and Last Name">
+<!ENTITY name.placeholder                  "Имя и фамилия">
 <!ENTITY email.accesskey                 "l">
 <!ENTITY email.placeholder                 "email@example.com">
 <!ENTITY password.label                  "Пароль:">
 <!ENTITY password.accesskey              "P">
-<!ENTITY password.placeholder              "Password">
+<!ENTITY password.placeholder              "Пароль">
 <!ENTITY password.text                   "Необязательно, будет использоваться только для проверки имени пользователя">
 
-<!ENTITY accountInformation.label        "Информация учётной записи">
-<!ENTITY outgoing.label                  "Исходящая:">
+<!ENTITY accountInformation.label        "Информация учетной записи">
+<!ENTITY outgoing.label                  "Исходящие:">
 <!ENTITY noEncryption.label              "Нет">
 <!ENTITY starttls.label                  "STARTTLS">
```

Comment:
**WRONG SUGGESTION**. STARTTLS - is a protocol name, Please see: **https://ru.wikipedia.org/wiki/STARTTLS**